src/imagewriter.cpp

```diff
@@ -1172,72 +1172,40 @@ void ImageWriter::_parseXZFile()
 
 bool ImageWriter::isOnline()
 {
-    return _online || !_embeddedMode;
-}
-
-void ImageWriter::pollNetwork()
-{
-#ifdef Q_OS_LINUX
     /* Check if we have an IP-address other than localhost */
     QList<QHostAddress> addresses = QNetworkInterface::allAddresses();
+    bool online = false;
 
     foreach (QHostAddress a, addresses)
     {
         if (!a.isLoopback() && a.scopeId().isEmpty())
         {
             /* Not a loopback or IPv6 link-local address, so online */
             emit networkInfo(QString("IP: %1").arg(a.toString()));
-            _online = true;
+            online = true;
             break;
         }
     }
 
-    if (_online)
-    {
-        _networkchecktimer.stop();
-
-        // Wait another 0.1 sec, as dhcpcd may not have set up nameservers yet
-        QTimer::singleShot(100, this, SLOT(syncTime()));
+    if (online) {
+        auto response = _networkManager.get(QNetworkRequest(QUrl(TIME_URL)));
```

Comment:
If `TIME_URL` is now only being used as an online connectivity-check, perhaps the comment at https://github.com/raspberrypi/rpi-imager/blob/qml/src/config.h#L13 ought to be updated too? :wink: